daisy/compute/compute.go

```diff
@@ -261,7 +275,15 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (Client, error)
 	if ep != "" {
 		rawBetaService.BasePath = ep
 	}
-	c := &client{hc: hc, raw: rawService, rawBeta: rawBetaService}
+	rawAlphaService, err := computeAlpha.New(hc)
+	if err != nil {
+		return nil, fmt.Errorf("alpha compute client: %v", err)
+	}
```

Comment:
What happens if alpha API is not enabled? Will this fail? 

Maybe we can we ignore the error and throw error on attempted use of alpha API so client (wrappers) can deal with it.

Reply:
Nothing bad will happen here. Client gets an error only when they actually call the alpha API.
If you look into the impl of computeAlpha.New, it always return a nil error, no real error will be returned, maybe because of the historic reason.

Reply:
This is my biggest concern. Most users will not have alpha api access, so that should be the default assumption and not throw unneeded errors. Just creating the client may not do that at all though as im pretty sure it doesn't make any network call, so thi might just be at api call time and then an error is fine.

Reply:
Yes Andrew as you said, error happens only when we call the API. It's as expected and we have the logic to properly handle the error.

Reply:
This line just copied what beta and GA client creation logic, even though it is not actually catching anything